src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,27 @@
+package org.example;
+
+import org.example.consumer.UserConsumer;
+import org.example.producer.UserProducer;
+
+public class Main {
+    public static void main(String[] args) {
+        String topic = "user-topic";
+        String schemaRegistry = "http://localhost:8081";
+        String kafkaServer = "localhost:9092";
+
+        UserProducer userProducer = new UserProducer(topic, kafkaServer, schemaRegistry);
```

Comment:
Optional: separate producer and consumer into 2 different apps and place them in distinguish Maven modules.